webgpu/webgpu-experimental.md

```diff
@@ -0,0 +1,80 @@
+# Experimental support for the WebGPU API
+
+The WebGPU API gives developers a low level, high performance, cross architecture way to program GPU hardware from JavaScript. It is the effective successor to WebGL on the Web. The spec has not yet been finalized, but support is currently being added to Firefox, Chromium, and Safari; and now also in Deno.
+
+This API gives you access to GPU rendering and general purpose GPU compute right from within Deno. Once finished, stabilized, and unflagged, this will provide a portable way to access GPU resources from web, server, and developer machine.
+
+## Examples
+
+Further examples outside of the ones listed below can be found in the webgpu-examples [repository](https://github.com/denoland/webgpu-examples).
+
+Here is a basic example that demonstrates accessing an attached GPU device, and reading out the name and supported features:
+
+```typescript
+/// Run with `deno run --unstable https://deno.com/v1.8/webgpu_discover.ts`
+
+// Try to get an adapter from the user agent.
+const adapter = await navigator.gpu.requestAdapter();
+if (adapter) {
+  // Print out some basic details about the adapter.
+  console.log(`Found adapter: ${adapter.name}`);
+  const features = [...adapter.features.values()];
+  console.log(`Supported features: ${features.join(", ")}`);
+} else {
+  console.error("No adapter found");
+}
+```
+
+This [example](https://github.com/denoland/webgpu-examples/tree/main/capture) shows how to capture an image by rendering it to a texture, copying the texture to a buffer, and retrieving it from the buffer.
+
+```typescript
+import {
+  copyToBuffer,
+  createCapture,
+  createPng,
+  Dimensions,
+} from "../utils.ts";
```

Comment:
this import seems causing type errors in CI check

Reply:
Just leaving a note for myself.

That import can be found here.
https://github.com/denoland/webgpu-examples/blob/main/utils.ts

Reply:
I think it's ok to import from `denoland/webgpu-examples` as it's officially maintained by denoland org. Maybe let's import from `https://github.com/denoland/webgpu-examples/blob/6c505182170ba10cc7bfe86203477688b96648bb/utils.ts`?

Reply:
error: Expected a JavaScript or TypeScript module, but identified a Unknown module. Importing these types of modules is currently not supported.
  Specifier: https://github.com/denoland/webgpu-examples/blob/6c505182170ba10cc7bfe86203477688b96648bb/utils.ts
    at file:///home/runner/work/manual/manual/webgpu/webgpu-experimental.md$39-80.ts:6:8
Error: Process completed with exit code 1.